client/signup/README.md

```diff
@@ -103,8 +103,10 @@ Note that here `apiRequestFunction` calls an API endpoint (`someRequest` in this
 
 The above example includes an inline function definition, but we should keep the `apiRequestFunction` values in `StepActions` (`signup/config/step-actions.js`) and include them like:
 
-```js
-apiRequestFunction: stepActions.createSite
+```javascript
```

Comment:
TIL `javascript` is not the same than `js` in this context.
`js` will remove `apiRequestFunction` from this code block because it think it is an unused label. `javascript` won't touch it.